CHANGELOG.md

```diff
@@ -1,12 +1,38 @@
+
 # nf-core/methylseq
 
-## v1.6dev - [date]
+## v1.6dev - [2020-02-15]
 
 ### Pipeline Updates
 
 * Updated template to tools 1.11
 * Moved parameter documentation into new `nextflow_schema.json` file.
 * Added new `--maxins` and `--minins` parameters to pass on to Bismark
+* Added Picard CollectInsertSizeMetrics and Picard CollectGcBiasMetrics
+* Improved qulimap and preseq by adding `samtools sort` and `samtools index` step in the Bismark aligner
+* Added BISCUIT aligner as an optional aligner, with all relative steps (alignment, mark duplicates with [samblaster](https://github.com/GregoryFaust/samblaster), methylation extraction, QC for biscuit, and optional [Epi-read](https://huishenlab.github.io/biscuit/epiread_format/) file creation with SNP information ).
+
+### Software updates
+
+* _new_: samblaster `0.1.26`
+* _new_: bedtools `2.30.0`
+* _new_: biscuit `0.3.16.20200420`
+* _new_: bcftools`1.10`
+* _new_: parallel `20201122`
+* _new_: gawk `5.1.0`
+* python `3.7.3` > `3.8.5`
+* markdown `3.1.1` > `3.3.3`
+* pymdown-extensions `6.0` > `8.1.1`
+* pygments `2.6.1` > `2.7.4`
+* pigz `2.3.4` > `2.5`
+* trim-galore `0.6.5` > `0.6.6`
+* samtools `1.9` > `1.10`
+* bowtie2 `2.3.5` > `2.4.2`
+* hisat2 `2.2.0` > `2.2.1`
+* bismark `0.22.3` > `0.23.0`
+* preseq `2.0.3` > `3.1.2`
+* picard `2.22.2` > `2.25.0`
+* methyldackel `0.5.0` > `0.5.1`
```

Comment:
I'd prefer to handle these separately if possible, to keep things clean. They're already being worked on in https://github.com/nf-core/methylseq/pull/185 so maybe let's try to merge that first and then see what's left in this diff.